CDCgov/prime-reportstream
# 7249 Support inline value sets in UP Sender Transform schema

This PR makes changes to the initial valueset implementation and adds a unit test to verify the valuesets function as expected.

Test Steps:
1. Run the `test transform with value set` unit test and verify the result passes.

## Changes
- Adjust the conditions in which valuesets are evaluated.
- Change the valueset value to be parsed as a string rather than a FHIRPath expression.
- Ensure the original untransformed value is preserved if a valueset key is not matched.
- Added unit testing for valueset.
- Reorganized unit tests and adjusted existing tests to match expected behavior.
- ktLint formatting updates unrelated to this story.

## Checklist

### Testing

## Linked Issues
- #7249
